apps/site/assets/css/_photo-gallery.scss

```diff
@@ -35,8 +35,12 @@
     display: none;
   }
 
+  .js & {
+    font-size: 0;
```

Comment:
What is this supposed to do?

Reply:
Instead of making each thumbnail link `display: block; float:left`, we're using `display: inline-block` which inherits the current font size, causing extra space to occur between wrapped lines. Setting font-size to zero eliminates this discrepancy, but only on browsers with JS enabled (non-JS browsers need font size to show the alt text for each thumbnail image).

Reply:
Gotcha - I'd add a tiny comment on this line to mention that. 

After tinkering with this a bit, I have some observations:

* Setting this to match the `<a>` margin you specified (.25%) instead of 0 also produces the desired outcome, and makes it a bit more clear what this is trying to achieve (match the spacing)
* This fix also works with line-height, which IMO makes more sense to change here than font-size (since there's no visible text here, and what we're looking at is essentially the line height of a line of images).